aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -102,6 +108,14 @@ module aptos_framework::account {
     const PROLOGUE_ESEQUENCE_NUMBER_TOO_BIG: u64 = 1011;
     const PROLOGUE_ESECONDARY_KEYS_ADDRESSES_COUNT_MISMATCH: u64 = 1012;
 
+    #[test_only]
+    const OFFER_ROTATION_CAPABILITY_SIGNATURE: vector<u8> = vector<u8>[128, 67, 218, 117, 250, 136, 201, 53, 214, 46, 129, 120, 102, 249, 108, 116, 158, 237, 0, 162, 227, 205, 107, 230, 159, 155, 102, 196, 166, 224, 135, 72, 138, 200, 37, 98, 140, 94, 109, 164, 48, 26, 239, 162, 140, 181, 202, 255, 87, 231, 63, 198, 50, 205, 207, 206, 89, 62, 58, 190, 116, 92, 179, 12];
+    #[test_only]
+    const OFFER_ROTATION_CAPABILITY_INVALID_SIGNATURE: vector<u8> = vector<u8>[100, 67, 218, 117, 250, 136, 201, 53, 214, 46, 129, 120, 102, 249, 108, 116, 158, 237, 0, 162, 227, 205, 107, 230, 159, 155, 102, 196, 166, 224, 135, 72, 138, 200, 37, 98, 140, 94, 109, 164, 48, 26, 239, 162, 140, 181, 202, 255, 87, 231, 63, 198, 50, 205, 207, 206, 89, 62, 58, 190, 116, 92, 179, 12];
+
+    #[test_only]
+    const OFFER_ROTATION_CAPABILITY_PUBLIC_KEY: vector<u8> = vector<u8>[246, 107, 240, 206, 92, 235, 88, 43, 147, 214, 120, 8, 32, 194, 2, 91, 153, 103, 174, 218, 162, 89, 189, 187, 159, 61, 2, 151, 236, 237, 14, 24];
```

Comment:
you can use something like x"hexdigits", see https://move-language.github.io/move/vector.html?highlight=hex%20string#vectoru8-literals